lib/modules/manager/html/readme.md

```diff
@@ -4,4 +4,4 @@ It also updates Subresource Integrity (SRI) hashes in `integrity` attributes.
 Key differences between the `cdnurl` manager and the `html` manager:
```

Comment:
Let's convert the list to a table.

I can't suggest the change, because of the removed lines. So here's the Markdown for you to copy/paste:

```
Key differences between the `cdnurl` manager and the `html` manager:

| Behavior                              | `html` manager | `cdnurl` manager                     |
| ------------------------------------- | -------------- | ------------------------------------ |
| Updating SRI hashes                   | Yes            | No                                   |
| Automatically finding files to update | Yes            | Must be given a `filePatterns` first |
```

Reply:
Unrelated to this RP , maybe in a separate one